win32ss/gdi/ntgdi/text.c

```diff
@@ -324,24 +322,23 @@ W32KAPI
 BOOL
 APIENTRY
 NtGdiGetTextExtentExW(
-    IN HDC hDC,
-    IN OPTIONAL LPWSTR UnsafeString,
-    IN ULONG Count,
-    IN ULONG MaxExtent,
-    OUT OPTIONAL PULONG UnsafeFit,
-    OUT OPTIONAL PULONG UnsafeDx,
-    OUT LPSIZE UnsafeSize,
-    IN FLONG fl
-)
+    _In_ HDC hDC,
+    _In_reads_opt_(Count) LPCWSTR UnsafeString,
+    _In_ ULONG Count,
+    _In_ ULONG MaxExtent,
+    _Out_opt_ PINT UnsafeFit,
+    _Out_writes_to_opt_(Count, *UnsafeFit) PINT UnsafeDx,
+    _Out_ LPSIZE UnsafeSize,
+    _In_ FLONG fl)
 {
     PDC dc;
     PDC_ATTR pdcattr;
     LPWSTR String;
     SIZE Size;
     NTSTATUS Status;
     BOOLEAN Result;
-    INT Fit;
-    LPINT Dx;
+    ULONG Fit;
+    PULONG Dx;
```

Comment:
Note that since `TextIntGetTextExtentPoint` take these parameters as (pointer for Dx) to signed ints, I wonder whether `TextIntGetTextExtentPoint` can indeed return negative values there; if not it may be worth investigating to consistently change these to unsigned ints in that function (and in the other callers).

Reply:
Unified to `PINT`.